### ISSUE TITLE
9.0.2 devtool still works even with admin permissions revoked

### ISSUE THREAD

**Issue by MishkaFur** [StrangeLoopGames/EcoIssues#18451]
If an admin has access to a devtool and has the permissions revoked, they are still able to use the devtool.

Devtool should be locked down and not function unless player is admin class, especially as there is no way ingame to locate them all incase one gets put into a wrong stockpile and into the hands of a player

**Comment by D3nnis3n** [comment]
I disagree, actually. The usage of the devtool is useful for creative building and i want people with no admin rights to be able to do that. A different solution to the 'keep' it issue might be good.

**Comment by MishkaFur** [comment]
Could perhaps a server option for "Devtool lockdown Y/N" work so that creative building can still be done in your example Dennis or i guess if feasible just an admin command of "/manage remove devtool,all" which wipes it from all players inventory?
